aleph_proxy_config/main.py

```diff
@@ -26,13 +27,23 @@
 
 async def download_nodes() -> Dict:
     # Iterate over trusted hosts in case the first is unavailable
+    last_error = None
     for trusted_host in TRUSTED_HOSTS:
         url = trusted_host + PATH
-        async with aiohttp.ClientSession(timeout=aiohttp.ClientTimeout(30)) as session:
-            async with session.get(url) as response:
-                response.raise_for_status()
-                data = await response.json()
-                return data
+        try:
+            async with aiohttp.ClientSession(timeout=aiohttp.ClientTimeout(30)) as session:
+                async with session.get(url) as response:
+                    response.raise_for_status()
+                    data = await response.json()
+                    return data
+        except ClientResponseError as error:
+            logger.exception(f"Error downloading nodes from {trusted_host}")
```

Comment:
Maybe a warning is more appropriate? This is supposed to happen once in a while and only matters if all hosts are down.